### ISSUE TITLE
Tray icon doesn't show graph but a straight horizontal line (Recover old Process Hacker behaviour)

### ISSUE THREAD

**Issue by PerikiyoXD** [winsiderss/systeminformer#1588]
### Brief description of your issue

The tray icon doesn't show a graph but a horizontal line.
![imagen](https://user-images.githubusercontent.com/3116368/215861845-48c818fd-6e16-44a1-8d87-65beed50060c.png)

**I'm aware this is a nightly build and not a release version**

### Steps to reproduce (optional)

Install System Informer 3.0.5917.
Open it.
Look at the tray icon.
Should show something like the image on the description.

### Expected behavior (optional)

Show a graph of CPU usage.

### Actual behavior (optional)

Show a straight horizontal graph of CPU usage, seems to show the latest graph point.

### Environment (optional)

```shell
System Informer (nightly) 3.0.5917
Windows version: Microsoft Windows 10 IoT Enterprise LTSC - 10.0.19044
```

**Comment by PerikiyoXD** [comment]
Looks like #1443

**Comment by PerikiyoXD** [comment]
![imagen](https://user-images.githubusercontent.com/3116368/215884775-c626d5d6-e080-49ef-8d6e-5f3d9ee9dec8.png)
It looks like it's just a matter of a default setting, enabling "CPU History" shows the "old" behaviour.

**Comment by PerikiyoXD** [comment]
![imagen](https://user-images.githubusercontent.com/3116368/215885114-41968a8f-542c-4000-8d68-f2dd8518d744.png)
This is the option, in case anyone wonders.

**Comment by PerikiyoXD** [comment]
![imagen](https://user-images.githubusercontent.com/3116368/215885810-429a0057-885f-44a6-b9ad-257563c8e1bf.png)
Comparing it with Process Hacker...

**Comment by PerikiyoXD** [comment]
This looks like a defaults misconfiguration.

Process Hacker (old) has CPU history enabled by default.
System Informer (new) has CPU usage enabled by default.

Maybe use old behaviour as default?

**Comment by PerikiyoXD** [comment]
Probably this issue must be closed and creating another one with the "real issue" is the next step.

**Comment by PerikiyoXD** [comment]
Editing one character does recover the original behaviour from Process Hacker.

There goes 2 hours of my life on searching across the source code.

**Comment by PerikiyoXD** [comment]
Tiny comment, when saving (on closing the application), there's an extra pipe '|' appended to the end of the setting string. I guess this is fine and doesn't need any extra attention. 
```xml
<setting name="IconSettings">2|1|</setting>
```

**Comment by dmex** [comment]
> This looks like a defaults misconfiguration.

The settings have remained exactly the same:
![image](https://user-images.githubusercontent.com/1306177/216017220-1bc083ca-ef08-4aff-8766-3f19ac7e8936.png)

The same commit included additional icons breaking the default: https://github.com/winsiderss/systeminformer/commit/3f1572babb6616f49ba7fac35c1d55029fba562b

> Maybe use old behaviour as default?

Yes. The CPU history icon should be the default.

**Comment by PerikiyoXD** [comment]
> The settings have remained exactly the same:

It's a 1:1 setting replacement but the icon registration order changed and that's the troublemaker.

Fixes
---

1. I got it fixed by swapping lines 349 and 350:
https://github.com/winsiderss/systeminformer/blob/298d16acf4a8f3a0417a2d62f71c4216f72f392a/SystemInformer/notifico.c#L349-L350
so it looks like this:
```c
 PhNfRegisterIcon(NULL, PH_TRAY_ICON_ID_CPU_HISTORY, PhNfpTrayIconItemGuids[PH_TRAY_ICON_GUID_CPU_HISTORY], NULL, L"CPU &history", 0, PhNfpCpuHistoryIconUpdateCallback, NULL); 
 PhNfRegisterIcon(NULL, PH_TRAY_ICON_ID_CPU_USAGE, PhNfpTrayIconItemGuids[PH_TRAY_ICON_GUID_CPU_USAGE], NULL, L"CPU &usage", 0, PhNfpCpuUsageIconUpdateCallback, NULL); 
```

2. Also fixed it by changing one character on here
https://github.com/winsiderss/systeminformer/blob/0f3dd2de35fce4f6e0a04263e93068c3239037a3/SystemInformer/settings.c#L112
to
```c
    PhpAddStringSetting(L"IconSettings", L"2|1");
```

Either way, I think that just changing a character is a small and good enough fix, better than swapping two lines that mimics the structure order.

I can open a PR and push a fix once we agree on a fix.

**Comment by PerikiyoXD** [comment]
By structure order I mean this structure:

https://github.com/winsiderss/systeminformer/blob/298d16acf4a8f3a0417a2d62f71c4216f72f392a/SystemInformer/include/notifico.h#L6-L20

**Comment by dmex** [comment]
> Tiny comment, when saving (on closing the application), there's an extra pipe '|' appended to the end of the setting string. I guess this is fine and doesn't need any extra attention.

This is intentionally left blank (null) to improve performance. Each tray icon setting is comprised of 3 parts `ID|Flags|PluginName` but the last part is optional. The parser still requires 3 parts for each sequence for example `1|2|` is one icon and `1|2||2|2||3|2|plugin` is 3 icons. If you remove the extra | then you have to add a lot of extra code and complexity to the parser with worse performance etc...  

> I can open a PR and push a fix once we agree on a fix.

The default CPU history is fixed with #1589 👍
